src/App.js

```diff
@@ -1,13 +1,89 @@
 import React from 'react';
```

Comment:
If you write `import React from 'react';` as `import React, { Component } from 'react';` line 19 could be written as `class App extends Component {` since you imported a named import you wouldn't have to use that dot.syntax. Either way is fine, just showing alternate ways.